### ISSUE TITLE
The model converges slowly during training

### ISSUE THREAD

**Issue by xiyufeng2** [ViTAE-Transformer/ViTDet#13]
The model converges slowly during training .  I trained model on Visdrone dataset with pretrained weights for 100 epochs.Total loss is declined to 0.55 and the mAP is near 5.2, which is too low.
Is it normal to have such a result？？Thank you.
